docs/internal/contributing/README.md

```diff
@@ -124,25 +124,55 @@ To add a new error:
 
 ## Changelog
 
-When appending to the changelog, the changes must be listed with a corresponding scope. A scope denotes the type of change that has occurred.
+We use the title and labels of merged pull requests to auto-generate the changelog, so there is no need to manually update [`CHANGELOG.md`](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+There's a GitHub action available in the repository named [Changelog Generation](https://github.com/grafana/mimir/actions/workflows/changelog-gen.yml) that can be triggered manually to re-generate the changelog for any release.
+
+An active decision to include a change in the changelog needs to be taken for every pull request. There's a pull request check named `Changelog Checks` that enforces this rule. By adding or removing labels on the pull request or updating the pull request title, description, or both, the check is re-evaluated.
+
+### Skip changelog
+
+If you don't want to include your change in the changelog, you need to add a label named `changelog-not-needed` to the pull request.
+
+There are other labels that are automatically excluded from the changelog but are mainly applied by GitHub Actions, e.g. `dependency-update`, `vendored-mimir-prometheus-update`, `helm-weekly-release`.
+
+### Include in changelog
+
+To include a pull request in the changelog, add a label named `add-to-changelog` to the pull request. Then the following additional validation rules are checked:
 
-The ordering of entries in the changelog should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`.
+- The title must be formatted according to `<Area>: <Summary>` (both "Area" and "Summary" should start with a capital letter), and it will appear as a line in the changelog verbatim.
```

Comment:
I think a PR title may not necessarily make a good changelog entry. As others have suggested, I think a explicit section in the PR description may be a better option.

Reply:
PS. I just realized there was a whole comment above addressing this.